webapp/src/ts/modules/messages/messages-content.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="content-pane right-pane">
+
+  <div class="item-content empty-selection" *ngIf="loadingContent">
+    <div><div class="loader"></div></div>
+  </div>
+
+  <div class="item-content empty-selection" *ngIf="!selectedConversation?.messages[0] && !loadingContent">
+    <div translate>No message selected</div>
+  </div>
+
+  <div class="message-content-wrapper item-content" *ngIf="selectedConversation?.messages[0] && !loadingContent">
+    <div class="body">
+
+      <div id="message-header" class="name-header">
+        <mm-sender [message]="selectedConversation.contact"></mm-sender>
+      </div>
+
+      <div id="message-content">
+        <div class="loader" *ngIf="loadingMoreContent"></div>
+        <p class="loading-status" *ngIf="allLoaded" translate>No more messages</p>
+        <div>
+          <ul>
+            <li *ngFor="let message of selectedConversation.messages; trackBy: listTrackBy; let lastMessage = last" [ngClass]="{'outgoing': message.doc?.kujua_message, 'incoming': !message.doc?.kujua_message}">
+              <div class="marker" *ngIf="firstUnread && firstUnread.id === message.doc._id" translate>Unread below</div>
+              <span class="message-body" [class.last-message]="lastMessage">
+
+                <div class="data">
+                  <p>
+                    {{message.doc?.sms_message?.message}}
+                    <span *ngIf="message.doc?.kujua_message">
+                      {{message.doc?.tasks[0]?.messages[0]?.message}}
```

Comment:
It's possible for a message to have no tasks.

Reply:
Added extra check for tasks